charts/erzwingungshaft/templates/cronjob.yaml

```
@@ -0,0 +1,47 @@
+{{- $dot := . }}
+{{- range $module := .Values.modules -}}
+{{ $data := dict "dot" $dot "module" $module }}
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)